test/end-to-end.js

```diff
@@ -1185,3 +1204,9 @@ function readBuiltinPhasesFromSanbox() {
   app.lazyrouter(); // initialize request handling phases
   return app._requestHandlingPhases;
 }
+
+function getSandBoxApp() {
```

Comment:
We really should just getApp or whatever (the sandboxing is part of the impl and doesn't need to be explicit in the name at all IMO).